src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1321,7 +1321,7 @@ internal string EnsureProperPathSeparators(string path)
                 path = path.Replace(@"\", @"\\");
             }
 
-            if (path.IndexOf(' ') != -1)
+            if (path.IndexOf(' ') != -1 || path.IndexOf('\'') != -1)
```

Comment:
nit: use `IndexOfAny`. Also, is it worth checking other special characters? Or do we just want to fix these when people complain about them?

Reply:
I think we can take the tact of updating it when there is reports of other characters that it doesn't like. This is the first time I've seen this come up.